tasks/need_fixing_linters/README.md

```diff
@@ -0,0 +1,39 @@
+
+# [HOWTO] Fix the linter errors for your team
+
+## Context
+
+Right after `7.48` merge freeze [we enabled `revive` and `gosimple` linters](). To avoid breaking `main` they are only linting commits made after [f40667d](https://github.com/DataDog/datadog-agent/commit/f40667d3841c6339be0d00d53e54a4a63f43f11e).
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ Missing the link for "enabling `revive` and `gosimple`"

Reply:
Thanks ! Fixed in [6d20653](https://github.com/DataDog/datadog-agent/pull/19015/commits/6d2065315f7ba06acbcea87bde309fd014ca4c61).